packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -196,7 +198,7 @@ export class ExecutionEngineHttp implements IExecutionEngine {
       const serializedVersionedHashes = serializeVersionedHashes(versionedHashes);
       const parentBeaconBlockRoot = serializeBeaconBlockRoot(parentBlockRoot);
 
-      const method = "engine_newPayloadV3";
+      const method = ForkSeq[fork] >= ForkSeq.eip6110 ? "engine_newPayloadV6110" : "engine_newPayloadV3";
```

Comment:
is the EL exposing the functionality at this method? (for prototyping?)

Reply:
> is the EL exposing the functionality at this method? (for prototyping?)

Yes newPayloadV6110 and getPayloadV6110 are already implemented on Besu